### PR TITLE
feat: implement doc to missed sort

### PR DESCRIPTION
**Pull request by derevandal**
- Add sort a collection by multiple fields and different ordering (https://github.com/gridsome/gridsome/issues/614)
- Change `featured` => `title` to let more clear

### CODE REVIEW

**Comment by hjvedvik** [comment]
Thank you :)

File: docs/querying-data.md [hunk 1]
```diff
@@ -79,6 +79,19 @@ query Posts {
 }
 ```
 
+#### Sort a collection by multiple fields and different ordering
+
+```graphql
+query Posts {
+  allPost(sort: [{ by: "date", order: DESC }, { by: "title", order: DESC }]) {
```
Comment (hjvedvik) [inline docs/querying-data.md]:
```suggestion
  allPost(sort: [{ by: "date", order: DESC }, { by: "title", order: ASC }]) {
```
Just to show both order types. And the query example misses the ending ` ``` `.

Comment (derevandal) [reply docs/querying-data.md]: thanks o/

**Review by thetre97 — APPROVED** [review]

**Comment by thetre97** [comment]
Also closes #564

**Pull request merged**